tests/admin/controller/test_metadata_services.py

```diff
@@ -19,20 +22,40 @@
 
 class TestMetadataServices(SettingsControllerTest):
 
-    def test_metadata_services_get_with_no_services(self):
+    def test_process_metadata_services_dispatches_by_request_method(self):
```

Comment:
Adding this test lets me simplify most of the other tests in this class. Those tests can test process_get or process_post instead of process_metadata_services, and they don't have to worry about checking permissions.